homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -117,72 +118,76 @@ def _state_schema(state):
     return vol.Schema(schema)
 
 
+_PLATFORM_SCHEMA_BASE = ALARM_CONTROL_PANEL_PLATFORM_SCHEMA.extend(
```

Comment:
What's the point of this constant? It's only used once.

Reply:
It was a copy from another place for visibility but I can agree we don't need this one.